src/lib/common/clockFunctions.cpp

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
I guess that this functionas are not new, but a refactoring of the ones we aready have from a previous PR. Is that correct?